config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -1031,11 +1034,12 @@ presubmits:
         name: ""
         resources:
           limits:
+            # Using 6 CPU to speed up bazel build phase (4 is enough for the test itself)
```

Comment:
Why you're not setting the flush-mem flag for kubemark

Reply:
Mixed it up with `pull-kubernetes-e2e-containerd-gce`